### PR TITLE
fix(animation-frames-timing-function): replace frames() which is not a timing function

### LINKED ISSUE #145: No output is showing on Web animations API
As you provided the link to learn about the [Web animations API frames example](https://github.com/mdn/css-examples/blob/main/animation-frames-timing-function/index-waa.html)

The file I was trying to issue on [mdn/css-examples/animation-frames-timing-function/index-waa.html](https://github.com/mdn/css-examples/blob/main/animation-frames-timing-function/index-waa.html)

I was trying to explore animations and encountered this file, which was not showing the output as expected.

It was just showing the blank screen below

image

As I tried to figure out what the issue is about, I figured out some exceptions in the console window of the browser says

```
Uncaught TypeError: Failed to execute 'animate' on 'Element': 'frames(10)' is not a valid value for easing at 
index-waa.html:50:17
```

Attaching the console error below

image

### PR DESCRIPTION
**Pull request by madhuroshan**
### Description

In [mdn/css-examples/animation-frames-timing-function/index-waa.html](https://github.com/mdn/css-examples/blob/main/animation-frames-timing-function/index-waa.html)

In ```div.animate(keyframes,options)``` options object have easing function given as frames(10) but frames(10) is not a valid value of easing functions.

Can't find any easing function named ```frames``` in mdn docs . So I replaced 
frames with steps (we can add others like cubic-bezier and other easing functions)

changed `var easingFunctions = ["frames(10)", "steps(10)", "ease-in"];` to
 `var easingFunctions = ["steps(5)", "steps(10)", "ease-in"];`

Fixes #145

### CODE REVIEW

**Comment by estelle** [comment]
The index page reads:

> The "animation-frames-timing-function" directory contains a simple example that demonstrates the difference between the steps() timing function available for CSS animations and transitions, and the new frames() timing function. [Run the example live](http://mdn.github.io/css-examples/animation-frames-timing-function/). You can also find versions that show the same timing function [used with transitions](http://mdn.github.io/css-examples/animation-frames-timing-function/index-transitions.html), and used with the [Web Animations API](http://mdn.github.io/css-examples/animation-frames-timing-function/index-waa.html).

So this text is intentional. We may want to delete the whole example rather than edit it and the paragraph in https://github.com/mdn/css-examples/blob/main/README.md that mentions it.

Paging @chrisdavidmills, as the original author ... 6 years ago.

File: animation-frames-timing-function/index-waa.html [hunk 1]
```diff
@@ -23,37 +23,39 @@
...
       }
-
     </style>
   </head>
   <body>
-    <div class="div1"><p>transition-timing-function: frames(10)<p></div>
-    <div class="div2"><p>transition-timing-function: steps(10);<p></div>
-    <div class="div3"><p>transition-timing-function: ease-in;<p></div>
+    <div class="div1">
+      <p>transition-timing-function: frames(10)</p>
```
Comment (chrisdavidmills) [inline animation-frames-timing-function/index-waa.html:30]: `frames(10)` => `steps(5)`?

**Review by chrisdavidmills — COMMENTED** [review]
I am impressed that you think I'll remember work that I did 6 years ago ;-)

This is really bizarre — I'm pretty sure I'm not going mad, and that there was at one point a timing function called `frames()`. But I can't find a mention of it anywhere except on these pages I wrote. It must've been introduced experimentally and then quickly abandoned.

I don't mind having an updated example with two different steps functions, or just deleting the `frames()` example altogether. Either is OK, IMO.

File: animation-frames-timing-function/index-waa.html (hunk 1 above)
Comment (madhuroshan) [reply animation-frames-timing-function/index-waa.html:30]: Yes, Either is ok. I started out recently so I don't know that there was a ```frames()``` function ,as i couldn't find it anywhere. I forgot to change in the text in html.

Comment (caugner) [inline animation-frames-timing-function/index-waa.html:30]:
```suggestion
      <p>transition-timing-function: steps(5)</p>
```

Comment (caugner) [reply animation-frames-timing-function/index-waa.html:30]:
```suggestion
      <p>transition-timing-function: steps(5)</p>
```

**Review by caugner — APPROVED** [review]

**Comment by madhuroshan** [comment]
okay

**Comment by OnkarRuikar** [comment]
> We just need to reformat the code, waiting on a reply on [this comment](https://github.com/mdn/css-examples/pull/153#issuecomment-1779583256) to understand how to proceed.

For now, I suggest you get the formatted code from [this Prettier playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhAEwmGBPABzgAIALGAWwBsA+AHSlXOuKoEMoBzAXjpAQC0AVwDOfelGLEmcNpglSpqCnBhtiYUmwBOI1bxBCYAMwEAOPsQD0CxctXqobFQYBuASzgB3AhG0xLSFgEGAMvd0wYUm5MOA8wOAFwyNJLGwZFaRh3GCo4GgB1OAAjYg53CjZs6BFiAEEABQBJYmNtZzhauAAPZwI81Cts3PyMuxF8PNtM4ohMPGJgMcypSu1OdygkYgAGAG5lxQBfBkOpTHdXRbPFXxEc92ht7Th2bNc4A8kVqWSo7Z2AFIvj8pKQ4O5OORtgAmHY7AjdEGg4psMAAa042ggQigmGecEwyMyJygNwI12+PzuDyeZWKIggVCMnxuvwi-2IMIArAikWziDAIARtgBGPnElZ5YwwMUSgWQKh+bZeUg5VlU46HQYTPBTMaDcFyWyoWbzabSC5XMDsEQiAxW0XiU6apQEGgwdpQe7VKACbIUTacATGXG4R5bYgTOAEEQACm5AEpBu6bqh3SmLYMrVmrRpbfa+FaYc7XdJ3Z6OD6I-6KkGQ2Hfdto7G4+LE3tM2mM1ZU5rs5dc5d82w7Q7LgBmUs-dMer3V6C1wNcBtQcN02R6ASbTu9i1umhd-tWHMuzKoERgbTuAgwCQ3Vw6YiboMAMUbEdq3GIAG0+C34yTPgABpiH-eBW3bECwP4UdEk2PgAF0vgfJ90TgPBX3aFQv1-AVgGIP5SG2PggWg1EMSxHE8RIkAXkwSwjmA-DCI5YiYPFMiQFAijMWxXF8Rg4pmTgRjmLLZCzxWR9tGIK1cOwMAhBUWAADoAEchDgbQ8AAZVeOBcD8OoqCoOMi0uPgOykzJjD8Yg4xk4h3GIb99mc6Q5MuERVLyLgoj2ZyAGogsTSlQS81wRB-dxENU8pKngON0Mw7DOlApYy0yF8uG2HLOHfNdfWi2LxIi84hHaJtiF5eEyvK9Uqs-bYmigYxNhyPB6p+I5rLLUlzysS9r1vE0rDNPAJENShaG4kBhWK5BQB0bEvAaHQEBEZAQDYKgvDYPAtuAkBinaDFVF0joABlNjgZBPS047TrRdCYF0gg0SDe7tEekA9EDb7fvuLg8gARSECB4GQYxdr0Y6ACsRG6XSgzBiGoaQGGqDhkBNMhuAGmxWNttHAQoDgQlCTmyt3CoIMAGEIAoSoSdMubgc4PI6hgT13GKFkGm0m7yeh2G4GO5gqAKNV4BED6En070Hg8fBtrAO05tcLTWtiWBdKvG8YDqPFdMmO7MbF46CGxPQCnaAhtutzptI+ObNj0fxCbYTgWYt7HxZAD7dG07bJbm63NhgAo2OQMwdmOl5NPcF4vZ9thRf9461GKaOUmQGFjtEOAABU2AZDOcbgChikJWJMCujhOCEb24FfPxEuyLgSaMCAQCOI4gA). 

Only change from default setting is `--bracket-same-line` is set to true.

**Comment by madhuroshan** [comment]
Add the formatted code and commit the changes?

**Comment by OnkarRuikar** [comment]
> Add the formatted code and commit the changes?

yes

**Review by caugner — APPROVED** [review]

**Comment by caugner** [comment]
Oops, sorry, hadn't see your last two comments!

**Comment by madhuroshan** [comment]
You have added the formatted code right? So, everything is done then?

**Comment by caugner** [comment]
> You have added the formatted code right? So, everything is done then?

Yes, all done. :)

**Comment by caugner** [comment]
Thank you @madhuroshan! 🎉

**Comment by madhuroshan** [comment]
Thank you @caugner @chrisdavidmills

**Pull request merged**